### PR TITLE
arranged assessment result screen

### PR DESCRIPTION
**Pull request by sangeetawinner**
Fixed assessment result folder structure

### CODE REVIEW

File: lib/presentation/container/assessment_result_container.dart [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      builder: (context, state) {
+        List<AssessmentModel> selectedAnswerSet =
+            state.answerSelectSttate.selectedQuestionAnswerSet;
+        return Scaffold(
+          appBar: AppBar(
+            backgroundColor: Colors.orange,
+            title: const Text("Assessment Result"),
+            automaticallyImplyLeading: false,
+          ),
+          body: AssessmentResult(selectedAnswerSet: selectedAnswerSet),
```
Comment (IrfanMoonraft) [inline lib/presentation/container/assessment_result_container.dart]: add a comma after selectedAnswerSet

File: lib/presentation/widgets/assessment_result.dart [hunk 2]
```diff
@@ -1,56 +1,44 @@
...
-                    itemCount: selectedAnswerSet[index].options.length,
-                    shrinkWrap: true,
-                    padding: EdgeInsets.all(5),
-                    scrollDirection: Axis.vertical,
+    return ListView.builder(
+      itemBuilder: (BuildContext context, index) {
+        return Card(
+          child: ListTile(
+            title: Text(selectedAnswerSet[index].question),
+            // title: Text("This is title"),
```
Comment (IrfanMoonraft) [inline lib/presentation/widgets/assessment_result.dart]: remove commented code

File: lib/route/route.dart [hunk 3]
```diff
@@ -19,7 +19,7 @@ class RouteGenerator {
     return {
       asessmentContainer: (BuildContext context) => AssessmentContainer(),
```
Comment (IrfanMoonraft) [inline lib/route/route.dart]: create screen file for AssessmentContainer

File: lib/presentation/container/assessment_result_container.dart (hunk 1 above)
Comment (sangeetawinner) [reply lib/presentation/container/assessment_result_container.dart]: https://github.com/sangeetawinner/moonraft_assessment_task/pull/1/commits/4d23d2315fa492f8677250beb4dd17833be3ebaa

File: lib/presentation/widgets/assessment_result.dart (hunk 2 above)
Comment (sangeetawinner) [reply lib/presentation/widgets/assessment_result.dart]: https://github.com/sangeetawinner/moonraft_assessment_task/pull/1/commits/8840a2f49d38d4158872e6a0df6f9d718f6005b3

File: lib/route/route.dart (hunk 3 above)
Comment (sangeetawinner) [reply lib/route/route.dart]: https://github.com/sangeetawinner/moonraft_assessment_task/pull/1/commits/b64039a315428d36de12ad8579b120787a15b661

**Review by IrfanMoonraft — APPROVED** [review]

**Pull request merged**